config/configmapprovider/provider.go

```diff
@@ -24,15 +26,13 @@ import (
 //
 // The typical usage is the following:
 //
-//		r := mapProvider.Retrieve("file:/path/to/config")
-//		r.Get()
+//		cfg, stopWatch, err := mapProvider.Retrieve(ctx, "file:/path/to/config", onChange)
```

Comment:
```suggestion
//		cfg, endWatch, err := mapProvider.Retrieve(ctx, "file:/path/to/config", onChange)
```

GitHub won't let me make a single suggestion here, but apply to the rest of this docblock.